tests/codebuild/run_canarytest.sh

```diff
@@ -19,6 +19,7 @@ function delete_tests {
     kubectl delete hyperparametertuningjob --all
     kubectl delete trainingjob --all
     kubectl delete batchtransformjob --all
+    kubectl delete model --all
```

Comment:
nit: it's cleaner to delete models after hostingdeployments, otherwise controller will attempt to create the model again and then immediately delete it.